tests/lib/newlib/newlib_api/testcase.yaml

```diff
@@ -0,0 +1,11 @@
+tests:
+  libraries.libc.newlib.posix.apis:
+    tags: clib newlib ignore_faults
+    filter: not CONFIG_XTENSA and TOOLCHAIN_HAS_NEWLIB == 1
+    min_ram: 64
+    extra_configs:
+      - CONFIG_POSIX_API=y
+  libraries.libc.newlib.no.posix.config:
+    tags: clib newlib no posix config ignore_faults
```

Comment:
I don't think tag "no" "posix" "config" should be added.

Reply:
Some APIs are invoked depends on CONFIG_POSIX_API not enabled. Should I change to another name?